autobuild/mac/codeQL/autobuild_mac_3_copy_files.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
See, this is where the current workflow design seems broken.

It should know this is a CodeQL run and not schedule a copy step.

Reply:
I removed the file, since that is not my focus at the moment.